File: Software Technologies - Exams/Software-Tech-Exam-2-Sept-2017-Old-Version/C# Solution/ShoppingList/Models/Product.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace ShoppingList.Models
{
    public class Product
    {
        [Key]
        public int Id { get; set; }

        public int? Priority { get; set; }

        [Required]
        public string Name { get; set; }

        public int? Quantity { get; set; }

        [Required]
        public string Status { get; set; }
    }
}

File: Software Technologies - Exams/Shopping_List_CU/JavaScript Solution/models/product.js
const Sequelize = require('sequelize');

module.exports = function(sequelize){
    let Product = sequelize.define('Product', {
        priority : {
            type : Sequelize.INTEGER,
            require: true,
            allowNull: false
        },
        name : {
            type : Sequelize.TEXT,
            require: true,
            allowNull: false
        },
        quantity : {
            type : Sequelize.INTEGER,
            require: true,
            allowNull: false
        },
        status : {
            type : Sequelize.STRING,
            require: true,
            allowNull: false
        },
    }, {
        timestamps : false
    });
    return Product;
};